src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -233,35 +237,89 @@ public Command intakeAndConveyorCommandScoreL1() {
         });
   }
 
-  public Command intakeToBirdhousePhase1() {
-    return startEnd(
-            () -> {
-              extendPivotCommand().initialize();
-              intakeCommand().initialize();
-              conveyorFeed().initialize();
-            },
-            () -> {
-              extendPivotCommand().end(false);
-              intakeCommand().end(false);
-              conveyorFeed().end(false);
-            })
-        .until(sensor.beamBroken());
+  public Command floorIntake() {
+    return runEnd(
+        () -> {
+          goToPivotPosition(kPivotExtendAngle);
+          intakeMotor.set(kIntakeSpeed);
+        },
+        () -> {}); // FIXME: runENd might need to be a start end
   }
 
-  public Command intakeToBirdhousePhase2() {
-    return retractPivotCommand()
-        .andThen(Commands.waitUntil(this::atSetpoint)) // FIXME: Fix debouncing if neccesary
-        .andThen(conveyorFeed().until(sensor.beamBroken().negate()));
+  public Command floorOuttake() {
+    return runEnd(
+        () -> {
+          goToPivotPosition(kPivotExtendAngle);
+          intakeMotor.set(-kIntakeSpeed);
+        },
+        () -> {});
   }
 
-  public Command intakeToBirdhouse() {
-    return intakeToBirdhousePhase1()
-        .andThen(intakeToBirdhousePhase2())
-        .withName("Intake Phase 1 and 2");
+  public Command humanPlayerIntake() {
+    return runEnd(
+        () -> {
+          goToPivotPosition(kcoralStation);
+          intakeMotor.set(kIntakeSpeed);
+        },
+        () -> {});
+  }
+
+  public Command algaeIntake() {
+    return runEnd(
+        () -> {
+          goToPivotPosition(kalgae);
+          intakeMotor.set(-kIntakeSpeed);
+        },
+        () -> {});
+  }
+
+  public Command algaeHold() {
+    return runEnd(
+        () -> {
+          goToPivotPosition(kalgae);
+          intakeMotor.set(-kIntakeSpeed);
+        },
+        () -> {});
+  }
+
+  public Command algaeOuttake() {
+    return runEnd(
+        () -> {
+          goToPivotPosition(kalgae);
+          intakeMotor.set(-kIntakeSpeed);
+        },
+        () -> {});
+  }
+
+  public Command l1ScoreModeA() {
+    return runEnd(
+        () -> {
+          goToPivotPosition(kl1);
+          intakeMotor.set(kIntakeSpeed);
+        },
+        () -> {});
+  }
+
+  public Command liScoreModeB() {
+    return runEnd(
+        () -> {
+          goToPivotPosition(kPivotRetractAngle);
+          intakeMotor.set(kIntakeSpeed / 5);
+          conveyorMotor.set(kConveyorSpeed);
+        },
+        () -> {});
+  }
+
+  public Command Idle() {
+    return runEnd(
+        () -> {
+          goToPivotPosition(kPivotRetractAngle);
```

Comment:
You should set the other motors to 0 in this Command since this is likely your default Command.